rootfs/api/models/app.py

```diff
@@ -869,3 +881,35 @@ def routable(self, routable):
         except KubeException as e:
             self._scheduler.update_service(self.id, self.id, data=old_service)
             raise ServiceUnavailable(str(e)) from e
+
+    def _update_application_service(self, namespace, app_type, port, routable=False, annotations={}):  # noqa
```

Comment:
I'm going to let @kmala answer that one but having `{}` as a default is generally a failure for the linter. Supposedly you are supposed to use `None` and then do other stuff internally... kinda funky